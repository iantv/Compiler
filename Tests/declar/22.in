struct person{
	char name[10];
	int age;
}people[100];