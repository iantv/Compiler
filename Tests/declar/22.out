people: array of 100 of
	struct person
		age: variable
			int
		name: array of 10 of
			char
struct person
	age: variable
		int
	name: array of 10 of
		char
